packages/cli-kit/src/private/node/ui/components/ConcurrentOutput.tsx

```diff
@@ -1,69 +1,43 @@
 import {OutputProcess} from '../../../../public/node/output.js'
-import {AbortController} from '../../../../public/node/abort.js'
-import {treeKill} from '../../../../public/node/tree-kill.js'
-import {handleCtrlC} from '../../ui.js'
+import {AbortSignal} from '../../../../public/node/abort.js'
 import {addOrUpdateConcurrentUIEventOutput} from '../../demo-recorder.js'
-import useAbortSignal from '../hooks/use-abort-signal.js'
 import React, {FunctionComponent, useCallback, useEffect, useMemo, useState} from 'react'
-import {Box, Key, Static, Text, useInput, TextProps, useStdin, useApp} from 'ink'
+import {Box, Static, Text, TextProps, useApp} from 'ink'
 import stripAnsi from 'strip-ansi'
 import figures from 'figures'
 import {Writable} from 'stream'
 
 export type WritableStream = (process: OutputProcess, index: number) => Writable
 
-export interface Footer {
-  shortcuts: Shortcut[]
-  subTitle?: string
-}
-
-export interface FooterContext {
-  footer?: Footer
-  updateShortcut: (prevShortcut: Shortcut, newShortcut: Shortcut) => void
-  updateSubTitle: (subTitle: string) => void
-}
-
-interface Shortcut {
-  key: string
-  action: string
-  syncer?: (footerContext: FooterContext, abortController: AbortController) => void
-  state?: {
-    [key: string]: unknown
-  }
-}
 export interface ConcurrentOutputProps {
   processes: OutputProcess[]
-  abortController: AbortController
+  abortSignal: AbortSignal
   showTimestamps?: boolean
-  onInput?: (input: string, key: Key, exit: () => void) => void
```

Comment:
is this a breaking change we want to assume right?

Reply:
Yes it's breaking, but I'm planning to work on whatever code is using `onInput` and migrate them to the new way of doing things after I merge this PR.